tutorials/sar-basics/index.md

```diff
@@ -0,0 +1,181 @@
+---
+title: SAR basics
```

Comment:
In one of the places where the term "SAR" is used (title, description, or the opening paragraph), please explain what it stands for.

Reply:
Ah, you do it on line 29. Still, maybe earlier is better?

Reply:
updated in title